spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/filter/factory/RequestRateLimiterGatewayFilterFactory.java

```diff
@@ -20,8 +20,8 @@
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.cloud.gateway.filter.GatewayFilter;
-import org.springframework.cloud.gateway.filter.ratelimit.KeyResolver;
-import org.springframework.cloud.gateway.filter.ratelimit.RateLimiter;
+import org.springframework.cloud.gateway.filter.redis.KeyResolver;
+import org.springframework.cloud.gateway.filter.redis.ratelimit.RateLimiter;
```

Comment:
We can't move these as that will break backward compatibility

Reply:
Ok I will move that back np.